src/main/java/io/github/thebusybiscuit/mobcapturer/MobAdapter.java

```diff
@@ -41,7 +39,7 @@
         List<String> lore = new LinkedList<>();
 
         lore.add("");
-        lore.add(ChatColor.GRAY + "Health: " + ChatColor.GREEN + DoubleHandler.fixDouble(json.get("_health").getAsDouble()));
+        lore.add(ChatColor.GRAY + "Health: " + ChatColor.GREEN + json.get("_health").getAsDouble());
```

Comment:
The value should be sanitized.
So please revert it back (well, use the new method in this case since the old one was removed).

```suggestion
        lore.add(ChatColor.GRAY + "Health: " + ChatColor.GREEN + NumberUtils.roundDecimalNumber(json.get("_health").getAsDouble()));
```